colorama/ansitowin32.py

```diff
@@ -18,7 +18,8 @@ def is_stream_closed(stream):
 
 
 def is_a_tty(stream):
-    return hasattr(stream, 'isatty') and stream.isatty()
+    is_pycharm = 'PYCHARM_HOSTED' in os.environ
```

Comment:
I think this will look better if we do:
`if 'PYCHARM_HOSTED' in os.environ: return True` and then leave the original return as it was. This is not important, though - what do you think?